test/column/editor.test.tsx

```diff
@@ -57,10 +57,13 @@ describe('Editor', () => {
   it('should commit changes and close editor when clicked outside', async () => {
     render(<EditorTest />);
     await userEvent.dblClick(getCellsAtRowIndex(0)[0]);
-    expect(screen.getByLabelText('col1-editor')).toHaveValue(1);
+    const editor = screen.getByLabelText('col1-editor');
+    expect(editor).toHaveValue(1);
     await userEvent.keyboard('2222');
     await userEvent.click(screen.getByText('outside'));
-    await waitForElementToBeRemoved(screen.queryByLabelText('col1-editor'));
```

Comment:
`waitForElementToBeRemoved` is quite flaky, on my machine at least.

Reply:
I noticed the same thing with vitest as well